docs/participate/devnet.md

```diff
@@ -1,10 +1,51 @@
 ---
-sidebar_position: 1
+sidebar_position: 1 
 ---
 # Devnet
 
-:::tip Devnet coming soon!
+:::tip Devnet Program is live!
 
-You can signup [here](https://airtable.com/appr3wKRsn4VoOvpi/pagxxT6QcL0OwGH0e/form), and we'll reach out to you once it goes live! 
+You can mint our Devnet slot from our [Mint page](https://devnet-mint.powerloom.dev/). Once minted, you can navigate to the [getting started](../build-with-powerloom/devnet/getting-started.md) section to kickstart your devnet journey.  
 
 :::
+
+## What is Powerloom Devnet? 
+
+Powerloom Devnet is a special development environment that allows developers to easily build and test data-driven applications on Powerloom. It provides a sandbox where you can experiment, learn, and create without impacting the main network.
+
+Devnet enables you to efficiently customize and analyze blockchain data to power your applications. Whether you're building a simple prototype or a complex applications, Devnet makes the development process smooth and straightforward.
```

Comment:
We should rewrite this IMO. 
```
Whether you're building a simple prototype or a complex applications, Devnet makes the development process smooth and straightforward.
```
How does it make the process smooth and straightforward?